shapash/webapp/utils/callbacks.py

```diff
@@ -0,0 +1,629 @@
+from typing import Optional, Tuple
+
+import numpy as np
+import pandas as pd
+import datetime
+from dash import dcc, html
+import dash_bootstrap_components as dbc
+
+
+def select_data_from_prediction_picking(round_dataframe: pd.DataFrame, selected_data: dict) -> pd.DataFrame:
+    """Create a subset dataframe from the prediction picking graph selection.
+
+    Parameters
+    ----------
+    round_dataframe : pd.DataFrame
+        Data to sample
+    selected_data : dict
+        Selected sample in the prediction picking graph
+
+    Returns
+    -------
+    pd.DataFrame
+        Subset dataframe
+    """
+    row_ids = []
+    for p in selected_data['points']:
+        row_ids.append(p['customdata'])
+    df = round_dataframe.loc[row_ids]
+    
+    return df
+
+
+def select_data_from_filters(
```

Comment:
This method has a complexity of 30 and 13 args.
Would it be possible to split it?